packages/suite-build/vite.config.mts

```diff
@@ -97,6 +97,10 @@ const alias = [
         find: 'vm',
         replacement: require.resolve('vm-browserify'),
     },
+    {
+        find: 'process',
+        replacement: require.resolve('process'),
```

Comment:
I've also tried `require.resolve('process/browser')` and it's correctly resolved as a full path to `node_modules/process/browser.js`. Any idea how to make it work, @vojtatranta?

Reply:
Try changing `bufferPolyfillPlugin` .

I thin there were problems in process because we patch the process as well and this was clashing with it.